src/app/integrations/intacct/intacct-shared/intacct-connection-form/intacct-connection-form.component.html

```
@@ -0,0 +1,32 @@
+               </div>
+             </div>
+             <div class="tw-mb-16-px">
+              <p class="tw-text-form-label-text-color tw-text-14-px tw-pb-8-px tw-pt-16-px">{{ 'intacctConnector.userIdLabel' | transloco }}<app-mandatory-field [noSpacing]="true"></app-mandatory-field></p>
+              <input [placeholder]="'intacctConnector.enterUserIdPlaceholder' | transloco: { userId: 'intacct.common.userId' | transloco }" type="text" id="userId" name="userId" formControlName="userID" [ngClass]="isReconnecting ? '' : 'tw-w-400-px'" class="custom-focus tw-border tw-border-gray-300 tw-rounded tw-px-3 tw-py-2" [ngClass]="[connectIntacctForm.controls.userID.invalid && connectIntacctForm.controls.userID.touched ? 'error-box' : 'normal-box', isReconnecting ? 'tw-w-full' : 'tw-w-400-px']">
+              <div *ngIf="connectIntacctForm.controls.userID.invalid && connectIntacctForm.controls.userID.touched" class="incorrect-credential tw-pt-2-px">
+                  {{ 'intacctConnector.invalidUserIdError' | transloco: { userId: 'intacct.common.userId' | transloco } }}
+              </div>
+            </div>
+             <div [ngClass]="isReconnecting ? 'tw-mb-20-px' : 'tw-mb-16-px'">
+               <p for="userPassword" class="tw-text-form-label-text-color tw-text-14-px tw-pb-8-px tw-pt-16-px">{{'intacct.common.userPassword' | transloco}}<app-mandatory-field [noSpacing]="true"></app-mandatory-field></p>
+               <input [placeholder]="'intacctConnector.enterUserPasswordPlaceholder' | transloco: { userPassword: ('intacct.common.userPassword' | transloco | lowercase) }" type="password" formControlName="userPassword" [ngClass]="isReconnecting ? '' : 'tw-w-400-px'" class="custom-focus tw-border tw-border-gray-300 tw-rounded tw-px-3 tw-py-2" [ngClass]="[connectIntacctForm.controls.userPassword.invalid && connectIntacctForm.controls.userPassword.touched ? 'error-box' : 'normal-box', isReconnecting ? 'tw-w-full' : 'tw-w-400-px']">
```

HTMLHint (1.5.0)

[error] Duplicate of attribute name [ [ngClass] ] was found.

(attr-no-duplication)